src/creature.h

```diff
@@ -53,6 +53,7 @@ struct FindPathParams
 
 static constexpr int32_t EVENT_CREATURECOUNT = 10;
 static constexpr int32_t EVENT_CREATURE_THINK_INTERVAL = 1000;
+static constexpr int32_t EVENT_CREATURE_PATH_INTERVAL = 100;
```

Comment:
IMO this should be configurable by the end user. I would suggest making it a config for config.lua, and using that variable where you have "200" hardcoded. I will also mark that location with this review.

Reply:
I was thinking the same thing but am trying to make sure I didn't add any extras that would prevent the PR from being able to be implemented.